doc/source/user_guide.rst

```diff
@@ -435,32 +435,59 @@ To display verbose output while running memcpy::
 
 .. _output_directory:
 
-Output Directory
-================
-
-The exact contents on the output directory will depend on configuration options
-used, augmentations enabled, etc.
-
-At the top level, there will be a ``run.log`` file containing the complete log
-output for the execution. The contents of this file is equivalent to what you
-would get in the console when using --verbose option.
-
-The output directory will also contain a ``results.json`` file that lists any
-overall run metrics and artifacts that have been collected during the run.
-Depending on the augmentations that were enabled there may be other results
-files in different formats, for example the ``csv``
-:ref:`output processor <output-processors>`. In addition you will find a subdirectory
-for each successful iteration executed with output and its own ``results.json``
-file for that specific iteration.
-
-If a job fails to complete for some reason, then the output directory for that
-job will be moved into a new directory called ``__failed``. If the job was
-running on a platform that supports android then WA will take a screen capture
-and UI dump from the device.
-
-Finally, there will be a ``__meta`` subdirectory. This will contain a copy of
-the agenda file used to run the workloads along with other configuration
-files that were used for the execution of that run.
+Output
+======
+
+The output directory will contain, subdirectories for each job that was run,
+which will in turn contain the generated metrics and artificats for each job.
```

Comment:
Art[ei]facts

Reply:
artifacts  -- the US spelling is used throughout.